test/Azure.Functions.Cli.Tests/E2E/StartTests.cs

```diff
@@ -313,6 +313,37 @@ public async Task start_dotnet_csharp()
             }, _output);
         }
 
+        [Fact]
+        public async Task start_dotnet_isolated_csharp_net9()
+        {
+            await CliTester.Run(new RunConfiguration
+            {
+                Commands = new[]
+                {
+                    "init . --worker-runtime dotnet-isolated --target-framework net9.0 --verbose",
+                    "new --template Httptrigger --name HttpTrigger --verbose",
+                    "dotnet add package Microsoft.Azure.Functions.Worker --version 1.23.0-preview1-20240723.1 --source https://azfunc.pkgs.visualstudio.com/e6a70c92-4128-439f-8012-382fe78d6396/_packaging/AzureFunctionsTempStaging/nuget/v3/index.json",
```

Comment:
Do we need the worker package also updated? I thought we need only Worker.Sdk from the feature branch.
Also leave a TODO comment here that this is temporary, and this will be removed after we release the official package.

Reply:
I forgot to remove the worker package update. Test has been updated!